docs/api-guide/routers.md

```diff
@@ -118,6 +118,26 @@ The above example would now generate the following URL pattern:
 
 * URL pattern: `^users/{pk}/change-password/$`  Name: `'user-change-password'`
 
+In the case you do not want to use the default name generated for your custom action, you can use the url_name parameter to customize it.
+
+For example, if you want to change the name of our custom action to `'user-change-password'`, you could write:
+
+    from myapp.permissions import IsAdminOrIsSelf
+    from rest_framework.decorators import detail_route
+    
+    class UserViewSet(ModelViewSet):
+        ...
+        
+        @detail_route(methods=['post'], permission_classes=[IsAdminOrIsSelf], url_name='change-password')
```

Comment:
I'm confused here - is this line incorrect or is `user-` prepended to the `url_name`?